### PR TITLE
Converted to an 'intable editor' instead of an external editor

### PR DESCRIPTION
**Pull request by wkleinheerenbrink**

### CODE REVIEW

File: pretty_inline/static/pretty_inline/css/baseInlineFormset.css [hunk 1]
```diff
@@ -145,4 +155,43 @@
...
+
+}
+
+.inline-group .change-form.original {
+    display: none;
+}
+
+.inline-group .change-form.original fieldset {
+    display: none;
+    height: 0;
```
Comment (jgadelange) [inline pretty_inline/static/pretty_inline/css/baseInlineFormset.css]: Why set height to 0? Seems unnecessary

File: pretty_inline/static/pretty_inline/css/baseInlineFormset.css [hunk 2]
```diff
@@ -145,4 +155,43 @@
...
+}
+
+.inline-group .change-form.active {
+    background: #F5F8FF;
+}
+
+.inline-group .change-form.active fieldset {
+    height: auto;
```
Comment (jgadelange) [inline pretty_inline/static/pretty_inline/css/baseInlineFormset.css]: If height is not set to 0, this is not necessary.

**Comment by jgadelange** [comment]
Looks great! A few (minor) remarks:
- I added inline comments to the diff
- I think one should be able to save the whole item when an inline is still open.
- When you get inline form errors:
  - The style of the pretty inline is ugly 
  - When you press 'OK' I think the error style should disappear.
- Stacked inline can't be saved anymore ('Please save (or cancel) the active editor first')
- Can't delete items in stacked & tabular inline
- Small gap between form and row it belongs to not pretty (I think) and I think there should also be a border on the left and right side of the form (as at the bottom): style

File: pretty_inline/static/pretty_inline/css/baseInlineFormset.css (hunk 1 above)
Comment (wkleinheerenbrink) [inline pretty_inline/static/pretty_inline/css/baseInlineFormset.css]: You're right. will change it.

**Pull request merged**